ui/pages/confirmations/confirmation/confirmation.js

```diff
@@ -507,8 +506,8 @@ export default function ConfirmationPage({
           </button>
         </div>
       )}
-      <div className="confirmation-page__content">
-        {templatedValues.networkDisplay ? (
+      <Box className="confirmation-page__content" paddingTop={4}>
+        {templatedValues.networkDisplay && !process.env.CHAIN_PERMISSIONS ? (
```

Comment:
Rather than hardcoding this here, could we instead just set `networkDisplay` based on the ENV within the `add-ethereum-chain` template?

Reply:
This was resolved but the code appears unchanged?

Reply:
We don't want to render anything when we have the feature flag on. So, I think it's fine to keep it this way and instead remove it entirely when we will remove the feature flag?